### ISSUE TITLE
After your last release, the plugin no long protect my website

### ISSUE THREAD

**Issue by engram-design** [verbb/knock-knock#66]
### Discussed in https://github.com/verbb/knock-knock/discussions/65

<div type='discussions-op-text'>

<sup>Originally posted by **theredvik** May 30, 2023</sup>
FYI,
After your last release 2.0.9, the plugin no long seems to protects my website or do anything in Craft 4 last version.
- I've clear cache,
- reinstall
- try changing password
- Connected with a VPN external to see and try accessing the website to see

and still enable not protected when everything is enabled.</div>

**Comment by engram-design** [comment]
@theredvik There wasn't a massive change in this release, only adding `Cache-Control` support.

Is your site a multi-site? This commit https://github.com/verbb/knock-knock/commit/ebe1a4fcf470b593806d8e6ba512d426921bed0e was really the only other change which _may_ have affected things. I'd really need to know your Knock Knock settings. Particularly, if they're set in the control panel, or via config file.

**Comment by splendidrob** [comment]
I'm also having this issue. Nothing happens when enabling knock knock. Settings via knock-knock.php:

```
<?php

return [
    '*' => [
      'enabled' => false,
    ],
    'staging' => [
        'enabled' => true,
    ],
    'dev' => [
        'enabled' => true,
    ],
];

?>
```

Also doesn't work if I delete the settings file and just use the control panel to enable instead.

**Comment by Tim-Wils** [comment]
I had to revert back to 2.0.8, the enabled true/false in the config seems to be broken in the newest release.

**Comment by arnout-poppr** [comment]
Having the same issue in the new v2.0.9.
Downgrading to v2.0.8 worked for me too.

**Comment by engram-design** [comment]
Should be fixed in [2.0.10](https://github.com/verbb/knock-knock/releases/tag/2.0.10)
